jscomp/others/js_bigint.res

```diff
@@ -1,3 +1,98 @@
 /*** JavaScript BigInt API */
 
-type t
+@val
+/**
+The special value "Not a Number". See [`NaN`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NaN) on MDN.
+*/
+external _NaN: bigint = "NaN"
+
+@val
+@scope("Number")
+/**
+Tests if the given value is `_NaN`
+
+Note that both `_NaN = _NaN` and `_NaN == _NaN` will return `false`. `isNaN` is
+therefore necessary to test for `_NaN`. Return `true` if the given value is
+`_NaN`, `false` otherwise. See [`isNaN`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN) on MDN.
+*/
+external isNaN: bigint => bool = "isNaN"
```

Comment:
I don't think this is necessary in bigint module. Unlike number, bigint operations (except comparison) cannot be mixed with other types and never produce NaN.

Reply:
It makes sense to me.